modules/webbrowser/CMakeLists.txt

```diff
@@ -84,6 +84,7 @@ set(HEADER_FILES
     include/modules/webbrowser/properties/boolpropertywidgetcef.h
     include/modules/webbrowser/properties/buttonpropertywidgetcef.h
     include/modules/webbrowser/properties/ordinalpropertywidgetcef.h
+	include/modules/webbrowser/properties/minmaxpropertywidgetcef.h
```

Comment:
Maybe it's just GitHub or should this document be clang-formatted? Seems like the intendation is a bit off on row 87, 111 and perhaps 143.

Reply:
Check, fixed